plugins/modules/proxmox_cluster_ha_groups.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2025, Markus Kötter <koetter@cispa.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_cluster_ha_groups
+
+short_description: Management of HA groups in roxmox VE Cluster
+
+version_added: "1.0.2"
```

Comment:
Features can only be added to new minor versions, not to bugfix versions. So pick the next expected minor release:
```suggestion
version_added: "1.1.0"
```